cli/debugfs_test.go

```diff
@@ -62,5 +62,5 @@ func TestExternalBinaryDependency(t *testing.T) {
 	assert.Contains(t, err.Error(), "`parted` binary not found on the system")
```

Comment:
Once we rename "on the system" to "on this machine" for errors on not found binaries, we should do it consistently for all binaries.

Iow, here `parted` should also be updated and we should look for more instances in the code base.

(if you take this suggestion, please do it as a separate commit)